source/_components/litejet.markdown

```diff
@@ -49,3 +43,26 @@ litejet:
   - 'LV Rel #'
   - 'Fan #'
 ```
+
+### Trigger
+
+LiteJet switches can be used as triggers too to allow those buttons to behave differently based on hold time. For example, automation can distinguish quick tap versus long hold.
+
+- **platform** (*Required*): Must be 'litejet'.
+- **number** (*Required*): The switch number to be monitored.
+- **held_more_than** (*Optional*): The minimum time the switch must be held before the trigger can activate.
+- **held_less_than** (*Optional*): The maximum time the switch can be held for the trigger to activate.
+
+The trigger will activate at the earliest moment both held_more_than and held_less_than are known to be satisfied. If neither are specified, the trigger activates the moment the switch is pressed. If only held_more_than is specified, the trigger will activate the moment the switch has been held down at least that time. If held_less_than specified, the trigger can only activate when the switch is released.
```

Comment:
I suggest you mark held_more_than and held held_less_than with \` around them to make them stand out like this `held_more_than`.